.travis.yml

```diff
@@ -103,6 +86,6 @@ install:
 - pip install numpy
 - pip install -e .
 script:
-- pip install pytest pytest-cov pytest-datadir coveralls
-- pytest --cov=landlab --cov-report=xml:$(pwd)/coverage.xml -vvv
+- pip install pytest pytest-cov pytest-datadir coveralls jupyter nbformat dask holoviews
```

Comment:
quick thought: since jupyter, dask, and holoviews are things that are not landlab, but are required to run the notebooks... how do you think we best indicate this to users?

Reply:
@kbarnhart I agree. I was thinking of doing something along those lines.

Reply:
cool. I'll just add it now since I'm working on fixing the notebook (mostly fixed now)

Reply:
@kbarnhart I've made some changes to address this. See what you think.